test/conformance/testdata/conformance.txt

```diff
@@ -203,6 +203,7 @@ test/e2e/node/events.go: "should be sent by kubelets and the scheduler about pod
 test/e2e/node/pods.go: "should be submitted and removed"
 test/e2e/node/pods.go: "should be submitted and removed"
 test/e2e/node/pre_stop.go: "should call prestop when killing a pod"
+test/e2e/scheduling/limit_range.go: "should create a LimitRange with defaults and ensure pod has those defaults applied"
```

Comment:
```
-test/e2e/scheduling/limit_range.go: "should create a LimitRange with defaults and ensure pod has those defaults applied"
+test/e2e/scheduling/limit_range.go: "should create a LimitRange with defaults and ensure pod has those defaults applied."
```

Reply:
I've opted for removing the full-stop from the test name, since most of the other test names don't have it, and it's easy to miss. Applied in this commit: https://github.com/kubernetes/kubernetes/pull/78921/commits/06e10c7e88a7c195d429a4bb92c2429163d26a51